src/components/feed/NodeDetails/PluginTitle.tsx

```diff
@@ -81,11 +92,15 @@ const PluginTitle = () => {
           <Title headingLevel="h3" size="xl">
             <span>{pluginName}</span>
           </Title>
-          <AiFillEdit
+          {isCreator ? (
```

Comment:
same advice as above, do you think you could add a tooltip here explaining why the edit feature is not available?

Reply:
Sure, I'll add that